velox/type/TimestampConversion.cpp

```diff
@@ -273,7 +273,7 @@ bool tryParseDateString(
 
```

Comment:
unrelated to this change, for L253, should we cap the year to kMaxYear if we are not returning false in this case? or is that expected?